core/src/main/java/org/elasticsearch/search/aggregations/BaseAggregationBuilder.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
can we maybe simplify this and just make `PipelineAggregationBuilder` a subclass of `AggregationBuilder` and remote that interface? that would be awesome